src/main/java/seedu/binbash/parser/Parser.java

```diff
@@ -159,4 +155,31 @@ private ListCommand parseListCommand(String[] commandArgs) throws InvalidCommand
             throw new InvalidCommandException(e.getMessage());
         }
     }
+
+    static int parseIntOptionValue(String argument, String option) throws ParseException {
+        try {
+            int intValue = Integer.parseInt(argument);
+            return intValue;
+        } catch (NumberFormatException e) {
+            throw new ParseException(option + " must be a number");
+        }
+    }
+
+    static double parseDoubleOptionValue(String argument, String option) throws ParseException {
+        try {
+            double doubleValue = Double.parseDouble(argument);
+            return doubleValue;
+        } catch (NumberFormatException e) {
+            throw new ParseException(option + " must be a number");
+        }
+    }
+
+    static LocalDate parseDateOptionValue(String argument, String option) throws ParseException {
+        try {
+            LocalDate dateValue = LocalDate.parse(argument, EXPECTED_INPUT_DATE_FORMAT);
+            return dateValue;
+        } catch (DateTimeParseException e) {
+            throw new ParseException(option + " invalid, required format: dd-mm-yyyy");
```

Comment:
Kindly help to edit the exception message.
```suggestion
            throw new ParseException(option + " is invalid. Required format: dd-mm-yyyy");
```

Reply:
Done.